src/tree-sitter/ExpressionBuilder.ts

```diff
@@ -130,12 +131,25 @@ function toString(s: string): string {
 function recordFromMatch<T extends string>(
```

Comment:
I suggest renaming the function to `glueDefinitionFromMatch`, and `RecordLink` to `GlueDefinition`